lib/Target/ExportSMTLIB/ExportSMTLIB.cpp

```diff
@@ -558,6 +558,14 @@ struct StatementVisitor
     return success();
   }
 
+  LogicalResult visitSMTOp(ResetOp op, mlir::raw_indented_ostream &stream,
+                           ValueMap &valueMap) {
+    if (op->getNumResults() != 0)
+      return op.emitError() << "must not have any result values";
```

Comment:
This should already be checked by the op verifier, so no need to check again here.